.npmrc

```diff
@@ -0,0 +1 @@
+workspaces-update = false
```

Comment:
to address this [issue](https://github.com/semantic-release/npm/issues/495) with `npm version` and workspaces.